go.mod

```diff
@@ -46,16 +40,11 @@ require (
 	github.com/writeas/nerds v1.0.0
 	github.com/writeas/saturday v1.7.2-0.20200427193424-392b95a03320
 	github.com/writeas/slug v1.2.0
-	github.com/writeas/web-core v1.2.1-0.20200813161734-68a680d1b03c
+	github.com/writeas/web-core v1.2.1-0.20210306215149-e05f572eb21f
```

Comment:
I think `web-core` should have a tagged release with the recently merged change. But we can test the fix first